doc/doxygen/headers/glossary.h

```diff
@@ -1476,7 +1476,7 @@ Article{JK10,
  *
  *
  * <dt class="glossary">@anchor workstream_paper <b>%WorkStream paper</b></dt>
- * <dd>The "%WorkStream paper" is a paper by B. Turcksin, M. Kronbichler and W. Bangerth
+ * <dd>The "WorkStream paper" is a paper by B. Turcksin, M. Kronbichler and W. Bangerth
```

Comment:
The percent sign had a purpose of escaping the link to the WorkStream class. Did that not work?